libs/payments/cart/src/lib/checkout.service.spec.ts

```diff
@@ -83,8 +85,13 @@ describe('CheckoutService', () => {
   let subscriptionManager: SubscriptionManager;
 
   beforeEach(async () => {
+    mockStatsd = {
+      increment: jest.fn().mockReturnValue({}),
+    } as unknown as StatsD;
```

Comment:
We want to avoid doing this style of mocking and instead use the `MockStatsDProvider`